.github/workflows/beam_PreCommit_Java_Iceberg_IO_Direct.yml

```diff
@@ -88,7 +94,7 @@ jobs:
       - name: run IcebergIO build script
         uses: ./.github/actions/gradle-command-self-hosted-action
         with:
-          gradle-command: :sdks:java:io:iceberg:build
+          gradle-command: :sdks:java:io:iceberg:build :sdks:java:io:iceberg:integrationTest
```

Comment:
So, this works for now. But when you add tests that are too big for the direct runner, I think you need another job. And i presume you would then pass in pipeline options that control the data volume so that you could re-use existing tests at higher scale.

Reply:
Will refactor in a future PR to include load test compatibility